.github/workflows/ci.yml

```diff
@@ -32,3 +32,15 @@ jobs:
       run: |
         cargo run -- bisect-rustc --help | grep "Examples:"
         cargo run -- --help | grep "Commands:"
+
+  fmt:
+    name: rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Run rustfmt check
+        run: |
+          rustup component add rustfmt
```

Comment:
Maybe use `dtolnay/rust-toolchain` here as well so it always uses stable rust?
```
    - name: Install rustfmt
      uses: dtolnay/rust-toolchain@stable
      with:
        components: rustfmt
```

Reply:
Sounds reasonable. I went ahead and merged the changes of #251 with this.

Reply:
Hm, my intention was to replace the raw `rustup` command with `dtolnay/rust-toolchain`'s `components` parameter. Although it probably does just call rustup internally so at this point it's mostly a question of cosmetics.

Reply:
Oh, I misunderstood. Yea, just using `components` seems nicer.